Frontend/updateTranslation.sh

```
@@ -0,0 +1,115 @@
+echo "Successfully downloaded the translations --------------------------------- ✓"
+
+TRANSLATED_STR="translatedString.csv"
+TRANSLATED_KEY_VALUE="translated_key_value.csv"
+FINAL_OUTPUT="final_output.csv"
+
+
+awk 'NR>1 && !/#VALUE!/' "$OUTPUT_FILE" > "$TRANSLATED_STR" 
+
+updateTypesFile() {
+
+    > "$TYPES_FILE_PATH"
```

Shellcheck (0.10.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)